tabify/tiRiskyPublicIPsCountTabify.js

```diff
@@ -0,0 +1,21 @@
+import isEmpty from "lodash/isEmpty";
+import union from "lodash/union";
+
+/*
+ *Returns the cardinality of unique public IPs coming from source and destination IPs terms aggregations
+ */
+export default class TiRiskyPublicIPsCountTabify {
+    
+    process(response) {
+        const aggregations = response && response.aggregations;
+        if (!isEmpty(aggregations)) {
+            const uniqueSrcPublicIps = aggregations.uniqueSrcPublicIps && aggregations.uniqueSrcPublicIps.riskySrcIps.buckets 
```

Comment:
why not checking if riskySrcIps is thruthy?

Reply:
done.